### DISCUSSION TITLE
"mantine", what's the story behind the name?

### DISCUSSION THREAD

**Discussion by kkirby** [mantinedev/mantine#1692] [Q&A] [answered]
As the title suggests, I'm curious why the project is called Mantine. Thanks!

**Comment by rtivital** [comment, marked as answer]
You can find all information about Mantine in the interview – https://console.substack.com/p/console-87?s=r

**Comment by kkirby** [comment]
Pretty cool. Thanks. Adding the direct answer here for anyone who just wants the answer.

>Where did the name for Mantine come from?

>> I’ve enjoyed Pokémon games since I was 5 years old, so I’ve named it after [Mantine](https://bulbapedia.bulbagarden.net/wiki/Mantine_(Pok%C3%A9mon)). I like the name and hope that Nintendo won’t force me to change it, like it happened with [jade (pug)](https://github.com/pugjs/pug/issues/2184).
